js/output/pjs/pjs-output.js

```diff
@@ -166,6 +166,9 @@ window.PJSOutput = Backbone.View.extend({
     build: function(canvas, enableLoopProtect, loopProtectTimeouts) {
         this.processing = new Processing(canvas, (instance) => {
             instance.draw = this.DUMMY;
+            if (this.options.logFullScreen) {
+              instance.println("FULLSCREENLOG");
+            }
```

Comment:
It looks like this may have been used for testing purposes.  Please remove before merging.

Reply:
This is on purpose - this is what triggers the full-screen mode in the ProcessingJS logger. I describe more in the PR: https://github.com/Khan/processing-js/pull/12/files
I can play around with other approaches as desired, however.